### ISSUE TITLE
Migrate build and release pipelines from DevOps to Github Actions

### ISSUE THREAD

**Issue by fgheysels** [arcus-azure/arcus#292]
Since all our build verifications are ran via GitHub actions, I think it makes sense to do this for our build and deploy pipelines as well.

**Comment by stijnmoreels** [comment]
Yes, definitely makes sense. But will need some investigation, as the current CI/Release are using pure Azure DevOps and this has to be translated to the language of GitHub actions.

**Comment by stijnmoreels** [comment]
Planning PTB next Friday to investigate in GitHub Actions. 👍
